subprojects/test-runner/instrumentation-tests/src/main/kotlin/com/avito/instrumentation/internal/InstrumentationArgsResolver.kt

```diff
@@ -0,0 +1,85 @@
+package com.avito.instrumentation.internal
+
+import com.avito.instrumentation.configuration.InstrumentationPluginConfiguration
+import com.avito.kotlin.dsl.getBooleanProperty
+import com.avito.runner.config.InstrumentationParameters
+import org.gradle.api.Project
+import java.util.Locale
+
+/**
+ * Instrumentation args consumed in `parseEnvironment()`
+ *
+ * TODO: Make stronger contract: MBS-7890
+ */
+internal class InstrumentationArgsResolver(
+    private val analyticsResolver: AnalyticsResolver,
+    private val buildEnvResolver: BuildEnvResolver,
+    private val reportResolver: ReportResolver,
+) {
+
+    fun resolvePluginLevelParams(
+        argsFromScript: Map<String, String>,
+        project: Project,
+        extension: InstrumentationPluginConfiguration.GradleInstrumentationPluginConfiguration,
+    ): Map<String, String> {
+        val args = mutableMapOf<String, String>()
+        args.resolveArg("planSlug", argsFromScript) {
+            "${project.name.capitalize(Locale.getDefault())}Android"
```

Comment:
`project.name` is not unique in a repository.
How about `project.path`?

Reply:
It's probably correct, but apps with same name in one repo?
Path is ok though, i'll do it